lib/sles4sap/sap_deployment_automation_framework/naming_conventions.pm

```diff
@@ -378,17 +379,14 @@ sub get_workload_vnet_code {
 
 =head2 get_sdaf_inventory_path
 
-    get_sdaf_inventory_path();
+    get_sdaf_inventory_path(config_root_path=>'/config/path', sap_sid=>'QAS');
 
 Returns full Ansible inventory filepath respective to deployment type.
+B<config_root_path> can be obtained from function B<get_sdaf_config_path>.
 
 =over
 
-=item * B<env_code>:  SDAF parameter for environment code (for our purpose we can use 'LAB')
-
-=item * B<sdaf_region_code>: SDAF parameter to choose PC region. Note SDAF is using internal abbreviations (SECE = swedencentral)
-
-=item * B<vnet_code>: SDAF parameter for virtual network code. Library and deployer use different vnet than SUT env
+=item * B<config_root_path>: SDAF config root path
```

Comment:
Reducing number of arguments. Full path can be obtained but other function `get_sdaf_config_path` by the caller.